### PR TITLE
JBIDE-25701 - ensure minishift default username is correct

### PR DESCRIPTION
**Pull request by robstryker**
Signed-off-by: Rob Stryker <rob@oxbeef.net>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

File: plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/MinishiftServiceManagerEnvironmentLoader.java [hunk 1]
```diff
@@ -68,7 +68,9 @@ public ServiceManagerEnvironment loadServiceManagerEnvironment(IServer server, b
 		}
 
 		try {
-			return new ServiceManagerEnvironment(merged);
+			ServiceManagerEnvironment env = new ServiceManagerEnvironment(merged);
+			env.setDefaultUsername("developer");
```
Comment (adietish) [inline plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/MinishiftServiceManagerEnvironmentLoader.java]: dare to extract this to a constant for improved visibility?

File: plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java [hunk 2]
```diff
@@ -37,6 +37,7 @@
 
 	int openshiftPort = 8443;
 	String openshiftHost = "https://10.1.2.2";
+	String defaultUser = "openshift-dev";
```
Comment (adietish) [inline plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java]: same here, a constant would help.
And then I dont get it, why do we have a hard-coded default, which we then overwrite?

**Comment by robstryker** [comment]
@jeffmaury  Are you sure you were working with a clean .minishift folder?  This works 100% as expected for me.

**Comment by adietish** [comment]
@jeffmaury @robstryker for me minishift is launched, an OpenShift connection for user "developer" and a docker connection is created. The freshly created openshift connection has a default project "My Project" available as requested in https://issues.jboss.org/browse/JBIDE-25701. To me things look fine.
@jkopriva dare to test it, please, ideally in Windows (since @jeffmaury is running Windows)?

File: plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/MinishiftServiceManagerEnvironmentLoader.java (hunk 1 above)
Comment (adietish) [reply plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/MinishiftServiceManagerEnvironmentLoader.java]: @robstryker dare to introduce a constant, please?

File: plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java (hunk 2 above)
Comment (adietish) [reply plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java]: @robstryker where is this being used? In my test user "developer" was used.

Comment (robstryker) [reply plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerEnvironment.java]: openshift-dev is the default credentials for CDK.  developer is for minishift.

**Pull request merged**